.circleci/config.yml

```diff
@@ -513,49 +475,82 @@ jobs:
           study_key: fon
 
   app-build:
+    description: Run build and store build artifacts to Google cloud storage
     working_directory: *ng_workspace_path
     executor:
       name: build-executor
     parameters:
       study_key:
         type: string
+      store_build:
+        type: boolean
+        default: true
```

Comment:
Store step is optional. This job is used in new `build-test-pr` workflow. We don't want to save build artifacts from PR commits to Google bucket.